test/data/migrations/9/environments.json

```diff
@@ -13,6 +13,7 @@
         "endpoint": "*/:var/a(b)?c/[0-9]{1,5}",
         "documentation": "Test documentation",
         "enabled": true,
+        "randomResponse": false,
```

Comment:
Please do not modify files in `test/data/migrations`, `test/data/import/new` and `test/data/import/old`